### PR TITLE
Adding F1 Score Formula

### PR DESCRIPTION
**Pull request by nakul-py**
**Closes** #143   

***This PR Adds F1 Score Formula to the Classification Section.***

****The Compiled Formula Look like as this:****
Screenshot from 2024-12-10 13-55-37

:)

### CODE REVIEW

File: book/3-classification.tex [hunk 1]
```diff
@@ -400,7 +400,20 @@ \subsection{F1-score}
...
-    [FORMULA GOES HERE]
+    \tikz
+        \node[inner sep=2pt, font=\Large] (a) {
+            {
+                $\displaystyle
+                F1 = \frac{2}{\textcolor{white!25!blue}{P}^{-1} + \textcolor{violet}{R}^{-1}} = \frac{2\textcolor{white!25!red}{TP}}{2\textcolor{white!25!red}{TP} + \textcolor{teal!60!green}{FP} + \textcolor{gray}{FN}}
+                $
+            }
+        };
+   \draw[-latex, white!25!blue, semithick] ($(a.south)+(-2.7,0.1)$) to[bend left=25] node[pos=1, left] {\scriptsize \textbf{\color{white!25!blue} precision}} +(-.6, -.5);
```
Comment (santiviquez) [inline book/3-classification.tex:411]: the \scriptsize is not standard in the book formulas. can you remove it

Comment (santiviquez) [reply book/3-classification.tex:411]: the colors here are not the standarized formula colors. please use the following:

% formula colors
\definecolor{nmlpurple}{HTML}{3B0280}
\definecolor{nmlcyan}{HTML}{00C8E5}
\definecolor{nmlred}{HTML}{DD4040}
\definecolor{nmlgreen}{HTML}{4EB046}
\definecolor{nmlyellow}{HTML}{E1BC29}

**Review by santiviquez — COMMENTED** [review]
Hi @nakul-py, thanks a lot for this. It looks nice, but there are some inconsistencies with the previous formulas. I left some inline comments, which are mostly about removing the \scriptsize and using the already declared color palette.

**Comment by nakul-py** [comment]
This commit will close this issue #143 

https://github.com/NannyML/The-Little-Book-of-ML-Metrics/pull/148/commits/9dee06105d2f1b3014090b74c3895e55ade6862b

Thanks for your feedback!

**Comment by santiviquez** [comment]
thanks for the changes :)

**Pull request merged**